next-tavla/app/(admin)/utils/index.ts

```diff
@@ -170,6 +170,14 @@ export function getFormFeedbackForError(e?: TError): TFormFeedback {
                 variant: 'error',
             }
         }
+        case 'file/invalid-input': {
+            return {
+                form_type: 'file',
+                feedback:
+                    'Filen du prøver å laste opp inneholder spesialtegn. Bruk kun bokstaver, tall, bindestrek, understrek og punktum',
```

Comment:
Husk punktum på slutten av setningen (....tror jeg)

Reply:
Burde vi endret litt ordlyd til å inneholde "filnavn" på noe vis? Synes dette også funker, så kan være personlig preferanse også 🤷‍♀️ 

F. eks: 
"Filnavnet inneholder spesialtegn. Bruk kun bokstaver, tall, bindestrek, understrek og punktum."
"Filnavnet er ugyldig. Bruk kun bokstaver, tall, bindestrek, understrek og punktum."

Reply:
Skulle vi vurdert å lage litt mer tilpassede feilmeldinger her?

Hvis brukeren laster opp en fil med tegnet `%` gir vi denne feilmeldingen:

```
Navnet på filen er ugyldig. Filnavnet kan ikke inneholde "%".
```

Hvis brukeren laster opp en fil med tegnene `%` og `!` gir vi denne feilmeldingen:

```
Navnet på filen er ugyldig. Filnavnet kan ikke inneholde "%,!".
```

Det blir kanskje litt vanskelig for brukeren å forstå 🤔